client/src/components/ChartJs.js

```diff
@@ -1,56 +1,53 @@
 import React, { useEffect, useRef } from 'react';
+import { useSelector } from 'react-redux'
 import Chart from 'chart.js';
 
-// Configuring chart parameters and pusing dummy data
+const ChartJs = () => {
 
-const chartConfig = {
-    type: "line",
-    // passing data to out chart
-    data: {
-        // labels for X axe
-        labels: [
-            "2020-02-28", 
-            "2020-03-2", 
-            "2020-03-4",
-            "2020-03-5",
-            "2020-03-9"
-            ],
-        datasets: [
-            {
-            // title of the dataset
-            label: "Patent temperature",
-            // Y axe color
-            borderColor: 'rgb(200, 30, 132)',
-            // Do not fill space bitween X and Y axes
-            fill: false,
-            // data for Y axe
-            data: [80, 70, 56, 90, 55],
-            }
-            ]
-        },
-    options: {
-        // Chart positioning on the page
-        layout: {
-            padding: {
-                left: 0,
-                right: 0,
-                top: 0,
-              bottom: 50
-            }
-        },
-    scales: {
-        yAxes: [{
-            // Range on the values bitween which Y axe will be streched despite of the dataset values
-            ticks: {
-                suggestedMax: 150,
-                suggestedMin: 0
-                    }
-                }]
-            }
-        }
-};
+// Extratring dummy data from the store 
+const dataSample = useSelector(state => state.dataSample);
```

Comment:
Not familiar with react-redux however seem to be an error in the ChartJs component. Here is some of the error, `Error: Uncaught [Error: could not find react-redux context value; please ensure the component is wrapped in a <Provider>]`

Link to complete build error:
https://travis-ci.org/github/COVID-19-electronic-health-system/Corona-tracker/builds/664043124?utm_source=github_status&utm_medium=notification

Reply:
In my understanding this error caused by lack of tests written for redux. In current test ChartJs did not wrap in Provider. It means when test tryin to render component there is no connection to redux store. And I try to use useSelector (redux hook) it fails because redux not exists in the context of this test.

Next step will be write redux test for this component.